interface/wx/thread.h

```diff
@@ -1036,8 +1036,8 @@ class wxThread
 
         @warning
             It is a good idea to explicitly specify a value as systems'
-            default values vary from just a couple of KB on some systems (BSD and
-            OS/2 systems) to one or several MB (Windows, Solaris, Linux).
+            default values vary from just a couple of KB on some systems (BSD systems)
```

Comment:
The BSD note is likely to be obsolete too... I think the entire warning could be just removed, at least I haven't encountered this problem since a very, very long time.